plugins/modules/cluster_drs_recommendations.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Ansible Cloud Team (@ansible-collections)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: cluster_drs_recommendations
+short_description: Apply Distributed Resource Scheduler (DRS) recommendations on VMware vSphere clusters
+description:
+    - Applies DRS recommendations on VMware vSphere clusters.
+author:
+    - Ansible Cloud Team (@ansible-collections)
+
+options:
+    cluster:
+        description:
+            - The name of the cluster to be managed.
+        type: str
+        required: true
+        aliases: [ cluster_name ]
+    datacenter:
+        description:
+            - The name of the datacenter.
+        type: str
+        required: true
+        aliases: [ datacenter_name ]
+
+extends_documentation_fragment:
+    - vmware.vmware.vmware.documentation
+'''
+
+EXAMPLES = r'''
+- name: Apply DRS Recommendations for Cluster
+  vmware.vmware.cluster_drs_recommendations:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter_name: datacenter
+    cluster_name: cluster
+    enable: true
```

Comment:
Is 'enable' supported parameter?
`FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/bin/python3"}, "changed": false, "msg": "Unsupported parameters for (vmware.vmware.cluster_drs_recommendations) module: enable. Supported parameters include: cluster, datacenter, hostname, password, port, proxy_host, proxy_port, username, validate_certs (admin, cluster_name, datacenter_name, pass, pwd, user)."}`

Reply:
Nope, i missed that in the example docs. Ill remove it. I dont see it anywhere else (plugin code, tests, etc), let me know if you do